### PR TITLE
styling for the home page

### PR DESCRIPTION
**Pull request by n-white**
Adding styling for the home page. Made it so that the placement scales with the window size of the browser

Still need to add the rock, paper, scissor icons to the top right of the screen.

login page

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -8,12 +9,6 @@ html, body, #root {
   display: flex;
 }
 
-.centered-container {
```
[the remark this answers is not in the document]
Comment (n-white) [reply src/index.scss]: oh you're right - thanks for catching. let's keep it.

File: src/styles/home_page.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
+$loginButton-height: 44px;
+
+.homePage {
+	background-color: #151b44;
```
[the remark this answers is not in the document]
Comment (n-white) [reply src/styles/home_page.scss]: yeah, great idea. i created a `variables` module where we can store standardized colors, font-sizes, etc.

File: src/components/HomePage.tsx [hunk 3]
```diff
@@ -58,6 +39,7 @@ const styles = StyleSheet.create({
 
   title: {
```
[the remark this answers is not in the document]
Comment (n-white) [reply src/components/HomePage.tsx]: i'm not entirely sure. I know @amhunt was doing some stuff with aphrodite. was finding aphrodite to be particularly cumbersome for the sign in page.

[the remark this answers is not in the document]
Comment (kerzhner) [reply src/components/HomePage.tsx]: @n-white Do you think we should start removing aphrodite from our code and consolidate all css in sass modules? My preference is to have all css/sass in one place.

[the remark this answers is not in the document]
Comment (n-white) [reply src/components/HomePage.tsx]: i would prefer that. i think aphrodite may be a bit difficult to use for the designs we have.

**Comment by kerzhner** [comment]
My vote is to start getting rid of Aphrodite

**Pull request merged**